src/commands/PublishCommand.js

```diff
@@ -124,7 +124,8 @@ export default class PublishCommand extends Command {
       : [packagesToPublish];
 
     if (!this.updates.length) {
-      callback(new Error("No updated packages to publish."));
+      this.logger.info("No updated packages to publish.");
+      callback(null, true);
```

Comment:
Per #887, we definitely need to pass `false` as the second argument to `callback()`, here. Otherwise it seems we attempt to make a git commit on no changes, and that's just wrong.